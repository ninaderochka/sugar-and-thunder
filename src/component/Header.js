import React from 'react';
import background from "./images/background.png";
import image from "./images/image.png";

export default function Header() {
  return (

    <div className="text-black font-normal " >
      <div className="absolute top-[126px] left-0 w-full h-full bg-cover" ><img alt="#" src={background}/></div>
    
      <div className="container mx-auto flex px-5 py-24 md:flex-row flex-col absolute">

        <div className="font-Poppins lg:flex-grow md:w-1/2 lg:pr-24 md:pr-16 flex flex-col md:items-start md:text-left md:mb-0 items-center mt-">
          
          <p className="text-black ml-36 text-3xl sm:text-3xl">WE ARE HERE TO</p>
          <p className="text-black ml-36 text-9xl ">HELP</p>         

          {/* button */}
          <div className=" ml-32 py-5 mt-12">
            <button type="button" className="text-2xl rounded bg-cyan-300 py-2 px-3">BOOK AN APPOINTMENT</button>
          </div>

         </div>

       <div className="lg:max-w-lg lg:w-full md:w-1/2 w-5/6 mr-16 mt-6">
         <img src={image} alt="#" />
       </div>

        </div>
    </div>
  )
}