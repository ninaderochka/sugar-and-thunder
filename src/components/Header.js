import React from 'react';
import { Link } from 'react-router-dom';
import Button from './Button';
import background from '../images/background.png';
import image from '../images/image.png';

export default function Header() {
  return (
    <div className="text-black font-normal h-screen w-screen">
      <img
        className="absolute bg-center imagee -z-10 w-full"
        alt="#"
        src={background}
      />

      <div className="container flex py-24 md:flex-row flex-col justify-center">
        <div className="font-Poppins lg:flex-grow md:w-1/2 lg:pr-24 md:pr-16 flex flex-col md:items-start md:text-left md:mb-0 items-center">
          <p className="text-black ml-36 text-3xl sm:text-3xl uppercase">
            We are here to
          </p>
          <p className="text-black ml-36 text-9xl ">HELP</p>

          {/* button */}
          <div className=" ml-32 py-5 mt-12">
            <Link to="/Booking">
              <Button value="BOOK AN APPOINTMENT" />
            </Link>
          </div>
        </div>

        <div className="lg:max-w-lg lg:w-3/5 md:w-1/2 w-5/6 mr-16 mt-6">
          <img src={image} alt="#" />
        </div>
      </div>
    </div>
  );
}
