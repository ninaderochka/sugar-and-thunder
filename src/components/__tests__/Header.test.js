import React from 'react';
import renderer from 'react-test-renderer';
import Header from '../Header';

test('Header component renders correctly', () => {
  const component = renderer.create(<Header />);
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});