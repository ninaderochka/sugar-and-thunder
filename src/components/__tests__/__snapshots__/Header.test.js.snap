// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component renders correctly 1`] = `
<div
  className="text-black font-normal h-screen w-screen"
>
  <img
    alt="#"
    className=" absolute bg-center imagee -z-10"
    src="background.png"
  />
  <div
    className="container flex py-24 md:flex-row flex-col justify-center"
  >
    <div
      className="font-Poppins lg:flex-grow md:w-1/2 lg:pr-24 md:pr-16 flex flex-col md:items-start md:text-left md:mb-0 items-center"
    >
      <p
        className="text-black ml-36 text-3xl sm:text-3xl uppercase"
      >
        We are here to
      </p>
      <p
        className="text-black ml-36 text-9xl "
      >
        HELP
      </p>
      <div
        className=" ml-32 py-5 mt-12"
      >
        <button
          className="undefined font-poppins font-normal text-black text-2xl uppercase w-fit px-6 py-2.5 bg-button-blue leading-tight rounded shadow-md hover:bg-button-blue/90 hover:shadow-lg focus:bg-button-blue/90 focus:shadow-lg focus:outline-none focus:ring-0 active:bg-button-blue/90 active:shadow-lg transition duration-150 ease-in-out"
          onClick={[Function]}
          type="button"
        >
          BOOK AN APPOINTMENT
        </button>
      </div>
    </div>
    <div
      className="lg:max-w-lg lg:w-3/5 md:w-1/2 w-5/6 mr-16 mt-6"
    >
      <img
        alt="#"
        src="image.png"
      />
    </div>
  </div>
</div>
`;
